### ISSUE TITLE
footer - accessibility -  'need to get in touch' link does not have a hover state + focus state hardly visable

### ISSUE THREAD

**Issue by Dan-KeduTesting** [Buckinghamshire-Digital-Service/hr-service-desk-alpha#172]
can hover and focus be the same as the other links in the footer please?

![footer_-_'need_to_get_in_touch'_link_does_not_have_a_hover_state_+_focus_state_hardly_visable](https://user-images.githubusercontent.com/2722522/65869007-6497f280-e371-11e9-9434-276312addf9f.png)
